<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_PROGRAMMER</class>
 <widget class="QWidget" name="GUI_PROGRAMMER">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="HexFile_LineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="HexFile_Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex File:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BrowseHexFile_Button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveReadData_Button">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>370</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Read Data</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="ReadData_Edit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BurnData_Button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>370</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Burn to Device</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ReadData_Button">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>370</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string>Read from Device</string>
   </property>
  </widget>
  <widget class="QLabel" name="ReadData_Label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Read Data:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="HexPreview_Edit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>421</width>
     <height>251</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="HexPreview_Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex Preview: [NN TT CC Addr Data]</string>
   </property>
  </widget>
  <widget class="QLabel" name="BurnMethod_Label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Burn Method:</string>
   </property>
  </widget>
  <widget class="QLabel" name="HexFormat_Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex Format:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="HexFormat_Combo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="RefreshPreview_Button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh Preview</string>
   </property>
  </widget>
  <widget class="QComboBox" name="BurnMethod_Combo">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="ReadAll_Radio">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Read All</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">readSelect</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="ReadAddr_Radio">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Read Address</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">readSelect</string>
   </attribute>
  </widget>
  <widget class="QLineEdit" name="ReadAddr_Edit">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>20</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="InstructionsText_Label">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>60</y>
     <width>121</width>
     <height>291</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Connect Pin XX to Pin XX and remove the cap for 5.

Be sure to set the Hex Format and Burn Method fields for reading as well.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Instructions_Label">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Instructions:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="readSelect"/>
 </buttongroups>
</ui>
